cryptus-app/components/market_overview/net_worth/time_interval/time_interval.module.scss

```diff
@@ -0,0 +1,42 @@
+@use "/styles/variables" as v;
+
+.icon {
+  position: relative;
+  background-color: #ffffff;
+  border-radius: 50%;
+  width: 50px;
+  height: 50px;
+  font-size: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  cursor: pointer;
+}
+
+.icon:hover {
+  top: -2px;
+  background-color: #ebe7e3;
+  color: #93918e;
+  opacity: 1;
+  visibility: visible;
```

Comment:
what does this do?

Reply:
When you click on the time_interval icon, there is a small color change, color has been updated